app/src/main/java/fr/free/nrw/commons/review/ReviewInterface.java

```diff
@@ -10,8 +10,8 @@
  * Interface class for peer review calls
  */
 public interface ReviewInterface {
-    @GET("w/api.php?action=query&format=json&formatversion=2&list=recentchanges&rcprop=title|ids&rctype=new|log&rctoponly=1&rcnamespace=6&rctag=android%20app%20edit")
-    Observable<MwQueryResponse> getRecentChanges(@Query("rcstart") String rcStart);
+    @GET("w/api.php?action=query&format=json&formatversion=2&generator=categorymembers&gcmtype=file&gcmsort=timestamp&gcmdir=desc&gcmtitle=Category:Uploaded_with_Mobile/Android&gcmlimit=50")
```

Comment:
Yes, I tested it beyond 50 too. I tried multiple values, starting from 10. Since this API call includes a continuation field, more data is fetched beyond this limit and the images do not repeat because of the database. However, I noticed that it took longer to verify from the database and display the image with a lower limit, possibly because the images get repeated before more data is fetched in the background. So I chose a value that does not take too long to display the images on the screen.

Reply:
Understood, thanks!
Would you mind adding a comment explaining this? Then I should be able to merge, sorry I had promised this already but I just realized that this may need some attention.
I think it is acceptable, as not so many people will perform a large number of reviews.
This is already a great improvement over the existing behavior.

Reply:
Sure. I'll just add a comment and update you.

Reply:
Done. I have added a detailed comment here and slightly modified the comment in the `ReviewHelper` too. Also, if even larger number of reviews are needed, then I think we will have to work on the database as matching every image identifier would take long.